### PR TITLE
Added graphs for Module 3 exercises.

### PR DESCRIPTION
**Pull request by JackC92**
Added graphs for Problem 8, 9, 10 of Module 3.

image
image
image
image
image

### CODE REVIEW

**Comment by siefkenj** [comment]
A couple of comments: can you label the figures for #8 on the graph itself? Perhaps make each one (S, T, T+S) a different color and make the labels matching colors.  The units on T+S are not uniform, can you make them uniform?

For #9, I like the idea of having new dots added in orange. Let's fill out all the orange dots that will fit on the grid. You can use a tikz loop for this, if you're daring enough.

**Comment by JackC92** [comment]
Updated the graphs based on the comments. I also changed a notation in Module 7 exercises, where `\vec n` and `\vec x` were used together to represent the normal vector.
image
image

File: book/modules/module3-exercises.tex [hunk 1]
```diff
@@ -245,6 +329,149 @@
...
+						\fill [WildStrawberry] (0,0) circle (2pt);
+						\fill [WildStrawberry] (1,-1) circle (2pt);
+						\fill [WildStrawberry] (1,0) circle (2pt);
+						\fill [WildStrawberry] (1,1) circle (2pt);
+						\fill [WildStrawberry] (2,1) circle (2pt);
+						\fill [WildStrawberry] (2,2) circle (2pt);
+						\fill [WildStrawberry] (3,3) circle (2pt);
+					\end{axis}
+				\end{tikzpicture}
+				\item The set $(S+S)+S$ and $S+(S+S)$ are equal since vectors are associative. This means that $S+S+S$ is well-defined and we can drop the parentheses, so the expression $S+S+S$ makes sense.
```
Comment (siefkenj) [inline book/modules/module3-exercises.tex:431]: -> `since vector addition is associative`

File: book/modules/module3-exercises.tex [hunk 2]
```diff
@@ -245,6 +329,149 @@
...
+						\fill [WildStrawberry] (3,3) circle (2pt);
+					\end{axis}
+					\foreach \x [evaluate=\x as \ymax using 2*\x] in {0,...,4} {
+						\foreach \y in {0,...,\ymax} {
+							\fill [orange] (\x,-2+\y/2) circle (2pt);
+						}
+					}
+				\end{tikzpicture}
+				
+				Notice that we obtain the orange points when we add additional $S$ to the set sum. Since we are considering the infinite sum $S+S+S+S+\cdots$, the process continues infinitely.
```
Comment (siefkenj) [inline book/modules/module3-exercises.tex:472]: -> `add an additional $S$`

File: book/modules/module7-exercises.tex [hunk 3]
```diff
@@ -106,8 +106,8 @@
 				\item  
 					If $\vec{n} = \mat{x\\y\\z}$ is a normal vector, then
 					\begin{align*}
-						\vec x \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
-						\vec x \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
+						\vec n \cdot \mat{0\\2\\2} &= 0  \quad\implies\quad 2y+2z = 0\\
+						\vec n \cdot \mat{1\\1\\1} &= 0 \quad\implies\quad x+y+z = 0.
```
Comment (siefkenj) [inline book/modules/module7-exercises.tex:110]: Put this back to `x`. Here `x` stands for `(x,y,z)` by convention.

Comment (JackC92) [reply book/modules/module7-exercises.tex:110]: Should I also change the `n` in the line above to `x`?

Comment (siefkenj) [reply book/modules/module7-exercises.tex:110]: Ah, I didn't read the question carefully. Instead, let's keep these as `n`, but change the variables to `n_x`, `n_y`, and `n_z`.

File: book/modules/appendix2.tex [hunk 4]
```diff
@@ -447,7 +537,37 @@
...
+				axis line style={latex-latex},
+				ticklabel style={font=\tiny,fill=white},
+				xlabel style={at={(ticklabel* cs:1)},anchor=north west},
+				ylabel style={at={(ticklabel* cs:1)},anchor=south west}
+			]
+			\end{axis}
+			\coordinate (A) at (2,1);
+			\coordinate (B) at (1,-1);
+			\draw [mygreen, thick, -] ($-0.5*(A)$) -- ($2*(A)$);
+			\draw [myorange, thick, -] ($(A)-3*(B)$) -- ($(A)+2*(B)$);
```
Comment (siefkenj) [inline book/modules/appendix2.tex:565]: You shouldn't need the `-` as an argument to the `\draw` command.

File: book/modules/appendix2.tex [hunk 5]
```diff
@@ -324,8 +375,43 @@
 
 	\begin{example}
 		Use row reduction to find the complete solution to $\systeme{x+y+z=1,y-z=2}$
-	
-		XXX Finish. Express answer both in vector form and as a set.
+		
+		The corresponding augmented matrix of the system is
```
Comment (siefkenj) [inline book/modules/appendix2.tex:379]: of -> for

File: book/modules/appendix2.tex [hunk 6]
```diff
@@ -324,8 +375,43 @@
...
+		\[
+			\begin{bmatrix}[ccc|c]
+			1 & 1 & 1 & 1\\
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+		\]
+		
+		The matrix is already in pre-reduced row echelon form, so we need to zero above each pivot.
```
Comment (siefkenj) [inline book/modules/appendix2.tex:387]: -> so we only need to

File: book/modules/appendix2.tex [hunk 7]
```diff
@@ -324,8 +375,43 @@
...
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+			\xrightarrow{\text{row}_1\mapsto\text{row}_1-\text{row}_2}
+			\begin{bmatrix}[ccc|c]
+			1 & 0 & 2 & -1\\
+			0 & 1 & -1 & 2
+			\end{bmatrix}
+		\]
+		
+		To solve the system
```
Comment (siefkenj) [inline book/modules/appendix2.tex:400]: + The third column is a free variable column, so we introduce ...

File: book/modules/appendix2.tex [hunk 8]
```diff
@@ -324,8 +375,43 @@
...
+		\[
+			\systeme{x+2z=-1,y-z=2}
+		\]
+		for $(x,y,z)$, we introduce the arbitrary equation $z=t$ and solve the system in terms of $t$.
+		
+		The complete solution to the system in vector form is
+		\[
+			\mat{x\\y\\z} = \matc{-1-2t\\2+t\\t} = t\mat{-2\\1\\1}+\mat{-1\\2\\0}
```
Comment (siefkenj) [inline book/modules/appendix2.tex:408]: -> `, and written as a set, the solution set is ...`

File: book/modules/appendix2.tex [hunk 9]
```diff
@@ -68,8 +93,34 @@
...
 		columns are pivot columns. For those that are not, what condition(s) fail?
-
-		XXX Finish include examples of 4 matrices, one of which is in RREF
+		\[
+			A=
+			\begin{bmatrix}[cccc]
+			1 & 0 & 0 & 2\\
+			0 & 0 & 0 & 0\\
+			0 & 0 & 1 & 7
+			\end{bmatrix},\,
```
Comment (siefkenj) [inline book/modules/appendix2.tex:102]: Use `\qquad` and no commas

File: book/modules/appendix2.tex [hunk 10]
```diff
@@ -68,8 +93,34 @@
...
+			D=
+			\begin{bmatrix}[cccc]
+			0 & 1 & 3 & 6\\
+			1 & 0 & 0 & 9\\
+			0 & 0 & 1 & 4\\
+			\end{bmatrix}
+		\]
+		
+		$C$ is in reduced row echelon form and column 1, 2, 3 are the pivot columns. The fourth property fails for $A$. The first and second property fails for $B$. The second and third property fails for $D$.
```
Comment (siefkenj) [inline book/modules/appendix2.tex:123]: Explain these in order, and in more detail (for example by highlighting in color the columns, etc that are incorrect). Keep in mind, this is the *very first* time students are seeing rref.

File: book/modules/module7-exercises.tex [hunk 11]
```diff
@@ -104,41 +104,41 @@
...
 					Row reduction yields
 					\[
 						\Rref(A) = \mat{1&0&0\\0&1&1},
 					\]
 					so complete solution in vector form is 
 					\[
-						\vec x = t\mat{0\\-1\\1}.
+						\vec n = t\mat{0\\-1\\1}.
 					\]
 					This means that any non-zero multiple of $\mat{0&-1&1}$ is a normal vector for this plane, so the normal form of the plane is
```
Comment (siefkenj) [inline book/modules/module7-exercises.tex:124]: `the normal form` -> `a normal form`

File: book/modules/module7-exercises.tex [hunk 12]
```diff
@@ -104,41 +104,41 @@
...
 
 					Row reducing, we have 
 					\begin{align*}
 						\mat{1&6&8\\2&0&2}&\rightarrow \mat{1&6&8\\0&-12&-14}\\
 						&\rightarrow \mat{1&6&8\\0&6&7}\\
 						&\rightarrow \mat{1&0&1\\0&6&7}\\
 					\end{align*}
-					which has nullspace equal to $\Span{\mat{6\\7\\-6}}$.
+					which has nullspace equal to $\Span\Set*{\mat{6\\7\\-6}}$.
 					Since $\mat{0\\0\\9}$ is a point on the plane, we then get that the normal form of the plane is \[\mat{6\\7\\-6}\cdot \left(\mat{x\\y\\z}-\mat{0\\0\\9}\right)=0.\]
```
Comment (siefkenj) [inline book/modules/module7-exercises.tex:139]: `the normal form` -> `a normal form`

**Pull request merged**